internal/http/services/appprovider/appprovider.go

```diff
@@ -49,6 +49,10 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+const (
+	idDelimiter string = ":"
```

Comment:
Can you please change `getStatInfo` to use this constant as well?

Reply:
sure, good point